pom.xml

```diff
@@ -795,8 +795,8 @@
                                         <exclude>com.google.code.findbugs:jsr305:*:*:compile</exclude>
                                         <!-- Log4J excluded in favor of log4j-over-slf4j and logback -->
                                         <exclude>log4j:log4j:*</exclude>
-                                        <!-- Ban log4j-core less than 2.15.0 due to Log4Shell vulnerability -->
-                                        <exclude>org.apache.logging.log4j:log4j-core:(,2.15.0)</exclude>
+                                        <!-- Log4j 2 log4j-core excluded in favor of log4j-to-slf4j routing to logback -->
```

Comment:
While it wasn't directly past of the jira, it's it worth updating log4j to 2.17.0 to address CVE-2021-45105 (a DoS vulnerability found after the release of 2.16.0)?